packages/synthchron-tauri/src/components/react-flow/DrawerWrapper.tsx

```diff
@@ -0,0 +1,27 @@
+import { Drawer, Toolbar } from '@mui/material'
+
+interface DrawerProps {
+  side: 'left' | 'right'
+  children: React.ReactNode
+}
+
+export const DrawerWrapper: React.FC<DrawerProps> = ({ children, side }) => {
+  const size = 300
```

Comment:
Should this maybe be a parameter, or change dynamically?

Reply:
Yeah, we were thinking of making the drawers resizable in the future so it _will_ change dynamically. In that case this line would be replaced by a `useState` call, so I did not bother moving the variable somewhere outside the component. When we add the resize it will have to control its own size either way